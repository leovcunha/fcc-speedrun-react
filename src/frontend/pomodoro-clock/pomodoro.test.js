import React from "react";
import renderer from "react-test-renderer";
import Pomodoro from "../pomodoro-clock";

describe("pomodoro", () => {
    it("renders without crashing", () => {
        const component = renderer.create(<Pomodoro/>).toJSON();
        expect(component).toMatchSnapshot();
    });
});
