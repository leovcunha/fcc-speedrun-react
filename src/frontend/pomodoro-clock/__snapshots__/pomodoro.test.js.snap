// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pomodoro renders without crashing 1`] = `
<div
  className="pomodoro jumbotron"
>
  <div
    className="row h1 justify-content-center"
  >
    Pomodoro Clock
  </div>
  <hr
    className="p-5"
  />
  <div
    className="row justify-content-around"
  >
    <div
      aria-label="Basic example"
      className="btn-group"
      role="group"
    >
      <button
        className="btn btn-danger"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="btn btn-danger"
        type="button"
      >
        Timer
      </button>
      <button
        className="btn btn-danger"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
    <div
      aria-label="Basic example"
      className="btn-group"
      role="group"
    >
      <button
        className="btn btn-danger"
        onClick={[Function]}
        type="button"
      >
        -
      </button>
      <button
        className="btn btn-danger"
        type="button"
      >
        Break
      </button>
      <button
        className="btn btn-danger"
        onClick={[Function]}
        type="button"
      >
        +
      </button>
    </div>
  </div>
  <div
    className="row timer justify-content-center"
  >
    25
                    :00
  </div>
  <div
    className="row h1 justify-content-center"
  >
    5
                    :00
  </div>
  <div
    className="row justify-content-center"
  >
    <button
      className="btn btn-danger"
      onClick={[Function]}
      type="button"
    >
      Start
    </button>
    <button
      className="btn btn-danger"
      onClick={[Function]}
      type="button"
    >
      Reset
    </button>
  </div>
</div>
`;
